style.css

```diff
@@ -0,0 +1,593 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  /* margin: 0; */
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: hsl(0deg 3% 71% / 10%);
+}
+
+.Header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 58px;
+  width: 375px;
+  align-items: center;
+  background-color: #ffff;
+  padding-top: 20px;
+  padding-left: 18px;
+}
+
+.mylogo {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  margin: 58px;
+  margin-left: 24px;
+  font-family: 'Poppines', sans-serif;
+  letter-spacing: 0;
+}
+
+.main-section {
+  background-image: url('./image/Mobile-Main_3.png');
+  background-size: cover;
+  background-color: #ffff;
+  background-repeat: no-repeat;
+  position: absolute;
+  height: 650px;
+  top: 0;
+  left: -34px;
+}
+
+.container {
+  height: 718px;
+}
+
+.main-header {
+  margin-top: 104px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+}
+
+.home-text {
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 12px;
+}
+
+/* LET’S CONNECT */
+.connect {
+  margin-top: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
```

Comment:
- [ ] Well done styllig your portfolio, it can look more professional if you align `Let's connect` call to qctio and the social media icons to the left with the same margin as the text and the headline.

Figma design:

<img width="321" alt="Screenshot 2023-06-10 at 6 02 06 PM" src="https://github.com/tes-nat/My_Portofolio/assets/53347966/cb6b42d2-e710-42ad-b1f7-5a0e9181abb9">

Your design:

<img width="370" alt="Screenshot 2023-06-10 at 6 03 39 PM" src="https://github.com/tes-nat/My_Portofolio/assets/53347966/ba91055d-0c89-4179-8993-6fd64c9edc02">